src/gui/UBThumbnailWidget.h

```diff
@@ -622,6 +622,7 @@ class UBDraggableLivePixmapItem : public UBDraggableThumbnailItem
         QTransform mTransform;
         QTimer updateTimer;
         int updateCount;
+        bool ignoreNextEvent{false};
```

Comment:
@letsfindaway Shouldn't we maintain a consistency on how data members are initialized ?

Reply:
My idea was: if I add some code, then I already try to stick to what we want for a refactored OpenBoard. And Initializing members in the header instead of each constructor is bad according to https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c48-prefer-in-class-initializers-to-member-initializers-in-constructors-for-constant-initializers. I won't change existing code in a bug fix, but try to use a better style for all things I add.

But to keep consistency I can also move this to the constructor. It's just a trade-off.

Reply:
Just tell me what you want to have here.

Reply:
I understand. I personally think it should remain consistent until we refactor things, but each pov is arguable. 

My main concern is more that I'm not sure curly braces should be preferred to `=` for a single argument initialization. Ok it is what is shown in the C++ core guidelines, but I find it un-intuitive, and to my knowledge it doesn't makes any difference in this case. If there are no good reason to prefer `{42}` to `=42`, please use the latter.

see : https://herbsutter.com/2013/05/09/gotw-1-solution